server/base/src/main/java/org/apache/accumulo/server/ServiceEnvironmentImpl.java

```diff
@@ -47,8 +44,7 @@ public Configuration getConfiguration() {
 
   @Override
   public Configuration getConfiguration(TableId tableId) {
-    return tableConfigs.computeIfAbsent(tableId,
-        tid -> new ConfigurationImpl(context.getTableConfiguration(tid)));
+    return new ConfigurationImpl(context.getTableConfiguration(tableId));
```

Comment:
With the changes in #3588 I think it may be good to keep the map.  However this map could be harmful in the case of a long lived plugin (like the balancer) that is going through lots of tables.  When tables are deleted this map would keep junk around.  So wondering if the map should be replaced with a caffine cache that expires unused entries.

Reply:
The issue I was having, that I thought was caused by this map, has been resolved in a test I was working on. I'm fine with closing this issue PR and issue.

Reply:
I can throw that into 3588